guard.cpp

```diff
@@ -9,13 +9,32 @@ using namespace openpower::guard;
 
 void guardList(bool displayResolved)
 {
+    bool display = false;
     auto records = getAll();
     if (!records.size())
     {
         std::cout << "No Records to display" << std::endl;
         return;
     }
-    std::cout << "ID       | ERROR    |  Type  | Path " << std::endl;
+    for (const auto& elem : records)
```

Comment:
why you need two for loop? you can simplify the code by using stringstream

Reply:
Simplified. Done